packages/in_app_purchase/test/billing_client_wrappers/billing_client_wrapper_test.dart

```diff
@@ -39,6 +39,59 @@ void main() {
     });
   });
 
+  // Make sure that the enum values are supported and that the converter call
+  // does not fail
+  group('enumValues', () {
+    test('-3', () async {
+      BillingResponseConverter converter = BillingResponseConverter();
+      converter.fromJson(-3);
```

Comment:
nit: Could you combine these different assertions into one test? These are different statements, but it's really testing that all the expected values are supported. So I think it makes sense to put them in one test case instead of having one per enum value.

Reply:
OK, done.